avro-annotations/src/main/java/io/micronaut/jsonschema/AvroSchemaSource.java

```diff
@@ -0,0 +1,23 @@
+package io.micronaut.jsonschema;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * An annotation that loads the Avro schema from the resources' folder.
+ * This annotation can be applied to classes.
+ */
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.TYPE)
+public @interface AvroSchemaSource {
+
+    /**
+     * Specifies the name of the Avro schema file to be loaded.
+     * like ("salamander-schema.avsc")
+     *
+     * @return the name of the Avro schema file.
+     */
+    String value();
```

Comment:
@andriy-dmytruk do you have something similar for json-schema? we need the schema at runtime to do proper serialization, but need some way to load it from src/main/resources

Reply:
Not really needed there. I think using resource loader and supporting `classpath:`, `file:` and `url:` approach sounds right.